### PR TITLE
178 return known repos

### PR DESCRIPTION
**Pull request by ggainey**
addresses #178

### CODE REVIEW

**Comment by ggainey** [comment]
Don't merge this yet - thinking about the use-case, returning all data about all repos is probably not the best choice. Changes incoming.

**Comment by ggainey** [comment]
Don't merge this yet - thinking about the use-case, returning all data about all repos is probably not the best choice. Changes incoming.

**Comment by ggainey** [comment]
Modified output - a 'naked' request returns just the list of labels that can be used to ask for more details. Example:

~~~
(178_return_known_repos) ~/github/ggainey-vmaas $ time curl -X GET http://127.0.0.1:8080/api/v1/repos/ | python -m json.tool
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   264    0   264    0     0    264      0 --:--:-- --:--:-- --:--:--  128k
{
    "repository_list": [
        "rhel-6-workstation-rpms",
        "rhel-7-workstation-optional-rpms",
        "rhel-7-server-optional-rpms",
        "rhel-6-server-rpms",
        "rhel-7-workstation-rpms",
        "updates",
        "rhel-7-server-rpms",
        "rhel-6-workstation-optional-rpms",
        "rhel-6-server-optional-rpms"
    ]
}

real	0m0.022s
user	0m0.021s
sys	0m0.012s
~~~

File: webapp/repos.py [hunk 1]
```diff
@@ -91,6 +91,7 @@ class RepoAPI(object):
     def __init__(self, repocache):
         self.cache = repocache
 
+    # pylint: disable=no-self-use
```
Comment (MichaelMraka) [inline webapp/repos.py]: more pythonic way to silence this check is 
```
@staticmethod
def is_empty(data):
...
```

**Comment by ggainey** [comment]
We're going to reject this PR - the feature it addresses is something we dont' want this service to be used for

**Pull request closed without merging**